src/styles/card.scss

```
@@ -0,0 +1,97 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card {
+  max-width: 200px;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid white;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $black-color;
+  }
+
+  &__product-code {
+    margin: 0;
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-color;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    font-size: 10px;
+    margin: 0;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__count-review {
+    margin: 0;
+    line-height: 16px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    align-items: center;
+    color:$grey-color;
```

Be attentive with all your code
```suggestion
    color: $grey-color;
```